src/audio/base_fw.c

```diff
@@ -570,6 +572,8 @@ static int basefw_set_large_config(struct comp_dev *dev,
 	switch (param_id) {
 	case IPC4_FW_CONFIG:
 		return basefw_set_fw_config(first_block, last_block, data_offset, data);
+	case IPC4_PERF_MEASUREMENTS_STATE:
+		return 0;
```

Comment:
Why does the IPC4_PERF_MEASUREMENTS_STATE case in this switch statement always return success (0)? According to the documentation, the driver can check this state via LargeConfigGet and change it with LargeConfigSet.

Reply:
This IPC has limited use for now, but it's used in test cases for systick. But now that you mention it I forgot to add default case returning error to the `set_perf_meas_state` function, so it won't always return 0 (I'll push changes soon).

Reply:
Since you're adding support for this IPC in a later commit, this one line can be skipped in this.